### PR TITLE
Update to ember-cli 2.6.2

### PR DESCRIPTION
**Pull request by cibernox**
I was about to submit a PR when I found that the ember-cli version was ancient, so this PR only updates ember-cli and a following one will fix the problem that initially brought me here.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
+  - "4"
 
 sudo: false
 
-addons:
-  apt:
-    sources:
-    - google-chrome
-    packages:
-    - google-chrome-stable
```
Comment (martndemus) [inline .travis.yml]: Can you restore the google chrome testing part of this?

File: testem.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
-{
+/*jshint node:true*/
+module.exports = {
   "framework": "qunit",
   "test_page": "tests/index.html?hidepassed",
   "disable_watching": true,
   "launch_in_ci": [
-    "Chrome"
+    "PhantomJS"
```
Comment (martndemus) [inline testem.js]: Same here, restore Chrome as testing browser please.

File: testem.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
   ],
   "launch_in_dev": [
+    "PhantomJS",
```
Comment (martndemus) [inline testem.js]: :fire: PhantomJS :fire:

File: .travis.yml (hunk 1 above)
Comment (cibernox) [inline .travis.yml]: I considered it, do you know if this was tested in chrome for any particular reason. The tests are super-simple, and it seemed a waste of time to download and install chrome in travis for some tests with no UI.

But I surely can, just double checking.

Comment (martndemus) [inline .travis.yml]: Because that's how we setup every DockYard project.

Comment (cibernox) [inline .travis.yml]: fair enough

**Comment by cibernox** [comment]
@martndemus Any idea why travis fails to install chrome? I've copied exactly what was there before.

**Comment by martndemus** [comment]
@cibernox change:

```
sudo: false
```

to

```
sudo: required
dist: trusty
```

**Comment by cibernox** [comment]
@martndemus that worked

**Pull request merged**